packages/account/src/utils/Types.ts

```diff
@@ -89,6 +88,14 @@ type ConditionalValidationProps = RequireAtLeastOne<
   "defaultValidationModule" | "signer"
 >;
 
+type ConditionalBundlerProps = RequireAtLeastOne<
```

Comment:
I see you already have conditional props. would there be a case when bundler is not used at all?
if someone is using only paymaster service for estimations and just building userops. 
@AmanRaj1608

Reply:
As we discussed yesterday, I believe bundler should be required as you cannot do much without it and it will also help us to abstract things like chain id in the future. Clients that currently create smart accounts without bundler will need to update the codebase.

Reply:
we should mention this in the docs after release